_posts/2024-12-19-cosmicqc.md

```diff
@@ -0,0 +1,317 @@
+---
+title: "coSMicQC: A step toward quality control and improvement of morphology profiles"
+author: Jenna Tomkinson
+member: jenna-tomkinson
+tags:
+  - opinions
+  - process
+  - software
+---
+# Let's tour the single-cell morphological galaxy with coSMicQC!
+
+Hi everyone! 🤩
+
+Now that I have a bit of bandwidth, I wanted to take the time to come back to go over something that I am very excited and passionate about:
```

Comment:
Consider removing the time-tense of this sentence (i.e. outside audiences may not understand "now" or "bandwidth" in the context of this work).

Reply:
I will remove the first part, but keep the same casual style, thanks for pointing out!